src/NodaTime.Test/YearMonthTest.Misc.cs

```diff
@@ -52,5 +56,53 @@ public void PlusMonths(int year, int month, int monthsToAdd, int expectedYear, i
             var expected = new YearMonth(expectedYear, expectedMonth);
             Assert.AreEqual(expected, yearMonth.PlusMonths(monthsToAdd));
         }
+
+        [Test]
+        [TestCaseSource(typeof(Cultures), nameof(Cultures.AllCultures))]
+        public void ToStringTest_Iso(CultureInfo culture)
+        {
+            var yearMonth = new YearMonth(2022, 1);
+            using (CultureSaver.SetCultures(culture))
+            {
+                Assert.AreEqual(yearMonth.ToString(culture.DateTimeFormat.YearMonthPattern, culture), yearMonth.ToString());
+                //Assert.AreEqual("xyz", yearMonth.ToString());
+            }
+        }
+
+        [Test]
+        [TestCaseSource(typeof(Cultures), nameof(Cultures.AllCultures))]
+        public void ToStringBclEquality(CultureInfo culture)
+        {
+            // The BCL *sometimes* use the genitive month names for year/month, even though I don't
+            // think it should. There may be more complexity here, but for the moment, we'll just skip the cultures we
+            // expect to fail. (We would definitely want to know if new cultures started failing.)
+            string[] expectedFailures = { "ast-ES", "ca-AD", "ca-ES", "ca-ES-valencia", "ca-FR", "ca-IT", "es-PE", "es-UY", "gl-ES", "oc-FR" };
+            if (expectedFailures.Contains(culture.Name))
+            {
+                return;
```

Comment:
(Guessing there's no way to flag a skipped test to the test runner, as you can in JUnit?)

Reply:
I don't *think* so (from within the test). It's feasible in xUnit, but I don't know of a way of doing so in NUnit. (We wouldn't want to mark this as skipped in the same way as some others anyway - it's more "infeasible" than "feasible but skipped".)